function_examples/token-hider/README.md

```diff
@@ -17,24 +17,27 @@ When you deploy your site to Netlify, you'll need to define the following [build
 ## Localhost
 
 ### 1. Installation
-
 Clone the repo, from the command line, go to `function_examples/token-hider` folder and run `npm install`.
 
 ### 2. Run Lambda Functions
-
-Type in the following script in your console:
 `API_URL="https://maps.googleapis.com/maps/api/geocode/json" API_TOKEN="your Google Maps API key" npm run lambda:dev`
+
```

Comment:
Something is wrong with this line. I think some explanatory text may have been accidentally removed at some point. I'd check with @depadiernos to see what his intention was here.

Reply:
(I'm guessing that at some point there was a step to add those two variables to a `.env` file before running the `npm run lambda:dev` line at the end of the line.)

Reply:
I didn't realize this command would work! I think it's worth telling the reader to run these lines in the command line.